client/api/user.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const models = require("./models");
+const router = express.Router();
+const authMiddleware = require("./utils");
+
+router.get("/", authMiddleware, async (req, res) => {
```

Comment:
non-blocking: prob already thought about it or are aware, prob wanna have a diff middleware for admin routes like users since only admins should be able to read all user data

Reply:
Yeah, mostly just been helpful for debugging, and I think we can honestly deprecate the user tables, since our identity information can be stored on auth0, and then tie that to the hackerProfile directly